styles/style.css

```diff
@@ -1,8 +1,160 @@
-/*
-Colors:
+* {
+    font-family: 'Montserrat', sans-serif;
+}
 
-Text: 1A1A1A
-Green: #00B172
-White: #FFF
+#nav-bar-all {
+    display: flex;
+    padding-bottom: 10px;
+}
+
+#nav-bar-all > img {
+    max-height:60px;
+    max-width: 40%; 
+    overflow: visible;
+    padding-top: 0;
+    padding-bottom: 0; 
+    padding-left: 25px;
+    padding-right: 40%
+}
+
+.nav-bar-ul {
+    padding-left: 160px;
+    display: flex;
+    justify-content: flex-end;
+}
+
+.nav-bar-ul > li {
+    font-size: 18px;
+    list-style-type: none;
+    padding: 5px 10px;
+}
+
+.nav-bar-ul a {
+    text-decoration: none;
+    color: #1A1A1A;
+}
+
+header {
+    background-image: url(../images/landing.jpg);
+    background-size: cover;
+    height: 700px;
+    color: white;
+    text-align: center;
+    margin: auto;
+}
+
+.header-text {
+    padding-top: 210px;
+}
+
+.header-text > h1 {
+    font-size: 65px;
+}
+
+.header-text > p {
+    width: 45%;
+    margin-left: 360px;
+    text-align: center;
+    font-size: 22px;
+    font-weight: 100;
+}
+
+section {
+     padding-bottom: 30px;
+}
+
+section > h2 {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    font-size: 30px;
+    margin-top: 50px;
+}
+
+#section-div {
```

Comment:
Because you are using an id, perhaps be more specific in the naming of it. What section is referred to?